pyfar/dsp/filter/__init__.py

```diff
@@ -19,10 +19,14 @@
 from .audiofilter import (
     allpass,
     bell,
+    high_shelf,
     high_shelve,
     low_shelve,
+    low_shelf,
     high_shelve_cascade,
-    low_shelve_cascade
+    high_shelf_cascade,
+    low_shelve_cascade,
+    low_shelf_cascade
```

Comment:
```suggestion
    low_shelf_cascade,
```
to make diffs easier and avoid merge conflict